src/aggregators/direct.jl

```diff
@@ -1,135 +1,199 @@
 mutable struct DirectJumpAggregation{T,S,F1,F2,RNG} <: AbstractSSAJumpAggregator
-  next_jump::Int
-  prev_jump::Int
-  next_jump_time::T
-  end_time::T
-  cur_rates::Vector{T}
-  sum_rate::T
-  ma_jumps::S
-  rates::F1
-  affects!::F2
-  save_positions::Tuple{Bool,Bool}
-  rng::RNG
+    next_jump::Int
+    prev_jump::Int
+    next_jump_time::T
+    end_time::T
+    cur_rates::Vector{T}
+    sum_rate::T
+    ma_jumps::S
+    rates::F1
+    affects!::F2
+    save_positions::Tuple{Bool,Bool}
+    rng::RNG
 end
-DirectJumpAggregation(nj::Int, njt::T, et::T, crs::Vector{T}, sr::T, maj::S, rs::F1, affs!::F2, sps::Tuple{Bool,Bool}, rng::RNG; kwargs...) where {T,S,F1,F2,RNG} =
-  DirectJumpAggregation{T,S,F1,F2,RNG}(nj, nj, njt, et, crs, sr, maj, rs, affs!, sps, rng)
+DirectJumpAggregation(
+    nj::Int,
+    njt::T,
+    et::T,
+    crs::Vector{T},
+    sr::T,
+    maj::S,
+    rs::F1,
+    affs!::F2,
+    sps::Tuple{Bool,Bool},
+    rng::RNG;
+    kwargs...,
+) where {T,S,F1,F2,RNG} =
+    DirectJumpAggregation{T,S,F1,F2,RNG}(nj, nj, njt, et, crs, sr, maj, rs, affs!, sps, rng)
 
 
 ############################# Required Functions #############################
 
 # creating the JumpAggregation structure (tuple-based constant jumps)
-function aggregate(aggregator::Direct, u, p, t, end_time, constant_jumps,
-                    ma_jumps, save_positions, rng; kwargs...)
-
-  # handle constant jumps using tuples
-  rates, affects! = get_jump_info_tuples(constant_jumps)
-
-  build_jump_aggregation(DirectJumpAggregation, u, p, t, end_time, ma_jumps,
-                          rates, affects!, save_positions, rng; kwargs...)
+function aggregate(
```

Comment:
I am very much not in favor of this type of reformatting. This is using an absolutely enormous amount of vertical space, and making it much harder to actually read the (very simple two-line) function (which now barely fits on my laptop screen). Is there not a rule that can wrap multiple arguments? The closing parenthesis also has a weird placement that gives the impression the function has ended...

Reply:
Are you using different settings than ModelingToolkit uses? It doesn't seem to reformat in this way... See https://github.com/SciML/ModelingToolkit.jl/blob/9cb6a4d139162ac59d7bda77d1e836b7500f4d61/src/systems/jumps/jumpsystem.jl#L111